simpler_setup/tools/strace_timing.py

```
@@ -279,22 +279,116 @@ def print_tpot_table(buckets, label_for_hid=None, stream=sys.stdout):
+    # HBG host-bind stages (a2a3 host_build_graph). Distinct from TMR device Orch.
+    "args": "simpler_run.bind.args",
+    "prebuilt": "simpler_run.bind.prebuilt",
+    "host_orch": "simpler_run.bind.host_orch",
+    "relocate": "simpler_run.bind.relocate",
+    "h2d_image": "simpler_run.bind.h2d_image",
+    "h2d_graph": "simpler_run.bind.h2d_graph",
+    "h2d_sm": "simpler_run.bind.h2d_sm",
+    "h2d_arena": "simpler_run.bind.h2d_arena",
 }
 
 # Per-round table columns, in print order. "Effective" is the orch∪sched merged
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `∪` (UNION). Did you mean `U` (LATIN CAPITAL LETTER U)?

(RUF003)